service/src/main/kotlin/fi/espoo/evaka/attendance/ChildAttendanceQueries.kt

```diff
@@ -73,6 +73,27 @@ fun Database.Read.getChildAttendance(
         .exactlyOneOrNull<ChildAttendance>()
 }
 
+fun Database.Read.getChildAttendancesOfDate(
+    childId: ChildId,
+    unitId: DaycareId,
+    date: LocalDate
+): List<ChildAttendance> {
+    // language=sql
+    val sql =
+        """
+        SELECT id, child_id, unit_id, (date + start_time) AT TIME ZONE 'Europe/Helsinki' AS arrived, (date + end_time) AT TIME ZONE 'Europe/Helsinki' AS departed
+        FROM child_attendance
+        WHERE child_id = :childId AND unit_id = :unitId AND date = :date
+        """
+            .trimIndent()
+
+    return createQuery(sql)
+        .bind("childId", childId)
+        .bind("unitId", unitId)
+        .bind("date", date)
+        .toList<ChildAttendance>()
+}
```

Comment:
Why aren't you using the new style binder here?
e.g.
```kotlin
    return createQuery<ChildAttendance> {
            sql(
                """
        SELECT id, child_id, unit_id, (date + start_time) AT TIME ZONE 'Europe/Helsinki' AS arrived, (date + end_time) AT TIME ZONE 'Europe/Helsinki' AS departed
        FROM child_attendance
        WHERE child_id = ${bind(childId)} AND unit_id = ${bind(unitId)} AND date = ${bind(date)}
        """
            )
        }
        .toList<ChildAttendance>()
}
```

I think it would be a more readable + less code

Reply:
Hmm yeah, copy pasted this from above, I'll change it 👍

Reply:
query fully removed for now